src/ohri-form-context.tsx

```diff
@@ -30,6 +30,8 @@ export interface EncounterContext {
   setEncounterLocation(value: any): void;
   initValues?: Record<string, any>;
   setObsGroupCounter?: any;
+  orders?: Array<OpenmrsEncounter>;
+  setOrders(value: any): void
```

Comment:
Any good reason for tracking orders through the context? I'm worried we are normalising abusing the form context this way.

Reply:
I will stick to using the state in the ohri form component instead